modules/adonisjs-crud-generator.yml

```diff
@@ -0,0 +1,13 @@
+name: crud-generator
+repo: shagital/adonisjs-crud-generator
+description: 'Adonisjs Admin Panel Generator is a package that helps you quickly scaffold your typical CRUD admin interfaces. It generates the admin panel code based on the existing (migrated) table in the database'
+icon: dynamodb.png
```

Comment:
Could you just remove the dynamodb logo? If I'm not mistaken your package has nothing to do with it?

Reply:
My bad. Added an icon now